apps/src/code-studio/levels/levelGroup.js

```diff
@@ -65,10 +65,9 @@ window.initLevelGroup = function (
   }
 
   var throttledSaveAnswers = throttle(
-    saveAnswers.bind(this, null, submitSublevelResults), 20 * 1000, {
-      leading: true,
-      trailing: true
-    });
+    subLevelId => {
+      submitSublevelResults(saveAnswers, subLevelId);
+    }, 20 * 1000);
```

Comment:
What used to happen is that we would bind saveAnswers to have a completeFn of null and a beforeHook of submitSublevelResults. We would then call this with a subLevelId. The flow was as follows:
- saveAnswers would see that it has a beforeHook and call beforeHook, passing it's main content (sendResultsCompletion) as a callback.
- beforeHook/submitSubLevelResults would then do it's thing, and eventually call it's completion param, which was sendResultsCompletion
- sendResultsCompletion would then call sendReport

What happens now:
- We create an arrow function that takes a subLevelId
- When this function is called, we call submitSubLevelResults and pass it saveAnswers as a callback
- when submitSubLevelResults is finished, it calls saveAnswers, which calls sendReport

This way the function flow should be easier to follow.

I removed the leading/trailing params as these are already the defaults.